### ISSUE TITLE
ImageId in product variant not created or updated

### ISSUE THREAD

**Issue by ashishpathak2111** [nozzlegear/ShopifySharp#50]
ImageId in product variant not created or updated

**Comment by nozzlegear** [comment]
Thanks for the report! I'll take a closer look at this as soon as possible.

**Comment by nozzlegear** [comment]
Alright, I've run some tests and confirmed that the `ImageId` doesn't get set in product variants, but right now I don't believe it's a problem with the library. I ran the same API calls manually through Fiddler and still couldn't get them to work. The only thing that *did* work was specifically creating a product image via the [Product Image API] and then using that image's id in the product variant.  Nothing else would work, so I'm not sure if this is a bug on Shopify's end, or if it's an undocumented "feature". 

Unfortunately there isn't a `ProductImageService` for ShopifySharp yet, [but I've opened an issue to get it in soon](https://github.com/nozzlegear/ShopifySharp/issues/52). 

All of that said, it could be that I missed something and there is in fact a bug in the lib, so if anybody else wants to take a crack at this I'd be happy to accept a pull request.
